Barotrauma/BarotraumaClient/ClientSource/Map/Lights/LightManager.cs

```diff
@@ -695,25 +696,18 @@ public void UpdateObstructVision(GraphicsDevice graphics, SpriteBatch spriteBatc
                 if (diff.LengthSquared() > 20.0f * 20.0f) { losOffset = diff; }
                 float rotation = MathUtils.VectorToAngle(losOffset);
 
-                //the visible area stretches to the maximum when the cursor is this far from the character
-                const float MaxOffset = 256.0f;
-                //the magic numbers here are just based on experimentation
-                float MinHorizontalScale = MathHelper.Lerp(3.5f, 1.5f, ObstructVisionAmount);
-                float MaxHorizontalScale = MinHorizontalScale * 1.25f;
-                float VerticalScale = MathHelper.Lerp(4.0f, 1.25f, ObstructVisionAmount);
+                // Cone starts at the opposite edge of the circle so it doesn't need to be as tall
+                const float coneHeightMultiplier = 2f;
+                const float scaleMultiplier = 1f;
 
-                //Starting point and scale-based modifier that moves the point of origin closer to the edge of the texture if the player moves their mouse further away, or vice versa.
-                float relativeOriginStartPosition = 0.1f; //Increasing this value moves the origin further behind the character
-                float originStartPosition = visionCircle.Width * relativeOriginStartPosition * MinHorizontalScale;
-                float relativeOriginLookAtPosModifier = -0.055f; //Increase this value increases how much the vision changes by moving the mouse
-                float originLookAtPosModifier = visionCircle.Width * relativeOriginLookAtPosModifier;
-
-                Vector2 scale = new Vector2(
-                    MathHelper.Clamp(losOffset.Length() / MaxOffset, MinHorizontalScale, MaxHorizontalScale), VerticalScale);
+                float scale = (1f - ObstructVisionAmount) * scaleMultiplier;
+                Vector2 circleOrigin = new Vector2(visionCircle.Width, visionCircle.Height) / 2f;
+                Vector2 coneOrigin = new Vector2(0f, visionCone.Height / 2f);
+                Vector2 coneScale = new Vector2(1f / visionCone.Width, scale / visionCone.Height * coneHeightMultiplier) * (losOffset.Length() + circleOrigin.X);
```

Comment:
This seems to change the behavior of the existing vision circle too. I think this needs some clarification: how and why does it change the vision circle too?